public/css/game.css

```diff
@@ -0,0 +1,79 @@
+@import "/static/css/variables.css";
+@import "/static/css/common.css";
+
+* {
```

Comment:
Don't global CSS-selectors since they might affect the navigation and footer later on. Add `margin: 0` to the elements that actually needs it.